test/interop/fortran/multidimArray/chapelProcs.chpl

```diff
@@ -0,0 +1,53 @@
+use ISO_Fortran_binding;
+
+proc toChapelArray(ref A: CFI_cdesc_t, param rank = 1) {
+  assert(A.rank == rank);
+  var D: domain(rank);
+  var dims: rank*range;
+
+  for i in 0..#rank {
+    dims(i+1) = A.dim[i].lower_bound..A.dim[i].extent /* by A.dim[i].sm ? */;
+  }
+
+  D = {(...dims)};
+  var AA: [D] int; // getChplType(A.type);
+  return AA;
+}
+
+export proc chpl_library_init_ftn() {
+  extern proc chpl_library_init(argc: c_int, argv: c_ptr(c_ptr(c_char)));
+  var filename = c"fake";
+  chpl_library_init(1, c_ptrTo(filename): c_ptr(c_ptr(c_char)));;
+  chpl__init_chapelProcs();
+}
+
+proc CFI_cdesc_t.this(idx:int...?rank) ref {
```

Comment:
Would you comment this method?  I think it's the array accessor, but I'm not certain.

Reply:
Right, it allows array accesses to the `CFI_cdesc_t` record. I added a header comment for it.